diff --git a/lib/types.d.ts b/lib/types.d.ts
index 3fbd75bc510688028c17b81331ae39a5583f1f6c..569482cf90a4639fb7d72e60f7ddf7272f35a342 100644
--- a/lib/types.d.ts
+++ b/lib/types.d.ts
@@ -123,7 +123,7 @@ export interface IAnnotation {
 }
 interface IRenderer extends Ace.VirtualRenderer {
     placeholderNode?: HTMLDivElement;
-    scroller?: HTMLDivElement;
+    scroller: HTMLDivElement;
 }
 export declare type IAceEditor = Ace.Editor & {
     renderer: IRenderer;
